tests/test_product_pages_api.py

```diff
@@ -0,0 +1,141 @@
+# SPDX-License-Identifier: GPL-3.0+
+import requests_mock
+
+from retasc.product_pages_api import ProductPagesApi
+
+PP_URL = "https://product_pages.example.com"
+PP_API = ProductPagesApi(PP_URL)
+test_prod = "rhscl"
+test_release = "rhscl-3-8"
+release_url = PP_API.api_url_releases()
+
+
+def request_assertion(prod_or_release, method_under_test, url, res, **kargs):
```

Comment:
This does not help the tests to be easier to write or read. It would be much cleaner if the caller itself calls the method_under_test and `assert`.